.github/workflows/ci.yml

```diff
@@ -192,7 +198,8 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Build package for PyPI
       run: |
-        python -m pip install pip hatch --upgrade
+        python -m pip install --upgrade uv
+        uv pip install --system --upgrade hatch
```

Comment:
Could probably replace this whole thing with

```
pipx run hatch build -t sdist -t wheel
```

Reply:
Yeah. Example:

```console
$ docker run --rm -ti python:3.12 /bin/bash
root@d6210c38b316:/# python -m pip --quiet install pipx
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
root@d6210c38b316:/# python -m pipx ensurepath
Success! Added /root/.local/bin to the PATH environment variable.

Consider adding shell completions for pipx. Run 'pipx completions' for instructions.

You will need to open a new terminal or re-login for the PATH changes to take effect.

Otherwise pipx is ready to go! ✨ 🌟 ✨
root@d6210c38b316:/# . ~/.bashrc 
root@d6210c38b316:/# git clone --recursive https://github.com/CoffeaTeam/coffea.git && cd coffea
Cloning into 'coffea'...
remote: Enumerating objects: 17727, done.
remote: Counting objects: 100% (817/817), done.
remote: Compressing objects: 100% (355/355), done.
remote: Total 17727 (delta 481), reused 655 (delta 394), pack-reused 16910
Receiving objects: 100% (17727/17727), 34.85 MiB | 45.99 MiB/s, done.
Resolving deltas: 100% (11949/11949), done.
root@d6210c38b316:/coffea# pipx run hatch build -t sdist -t wheel
────────────────────────────────────────────────────────────────────────────────────────────────── sdist ───────────────────────────────────────────────────────────────────────────────────────────────────
dist/coffea-2024.4.2.dev11+g8dc655e9.tar.gz
────────────────────────────────────────────────────────────────────────────────────────────────── wheel ───────────────────────────────────────────────────────────────────────────────────────────────────
dist/coffea-2024.4.2.dev11+g8dc655e9-py3-none-any.whl
root@d6210c38b316:/coffea# 
```